src/test/java/com/marizueva/laboratory/hw5/pages/HomePage.java

```diff
@@ -0,0 +1,41 @@
+package com.marizueva.laboratory.hw5.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class HomePage extends BasePage {
+
+    @FindBy(css = "a[href='#'] >.caret")
+    private WebElement openLoginDropdownButton;
+
+    @FindBy(id = "name")
+    private WebElement loginField;
+
+    @FindBy(id = "password")
+    private WebElement passwordField;
+
+    @FindBy(xpath = "//ul/li[@class='dropdown uui-profile-menu open']"
+            + "/descendant::span[text()='Enter']")
+    private WebElement enterCredentials;
+
+
+    public HomePage(WebDriver driver) {
+        super(driver);
+    }
+
+    public CommonPage getCommonPageElements() {
+        return new CommonPage(driver);
+    }
+
+    public void openSite() {
+        driver.navigate().to(siteUrl);
+    }
+
+    public void performLogin() {
+        openLoginDropdownButton.click();
+        loginField.sendKeys(defaultUser);
+        passwordField.sendKeys(defaultPassword);
+        enterCredentials.click();
```

Comment:
Why **username** and **password** do not implemented as method parameters?

Reply:
fixed